cmplxs/cmplxs.go

```diff
@@ -85,6 +85,22 @@ func Count(f func(complex128) bool, s []complex128) int {
 	return n
 }
 
+// Complex fills each of the elements of dst with the complex number
+// constructed from the corresponding elements of real and imag.
+// A panic will occur if the lengths of arguments do not match.
+func Complex(dst []complex128, real, imag []float64) []complex128 {
+	if len(dst) != len(real) || len(dst) != len(imag) {
+		panic("cmplxs: length of destination does not match length of the source")
```

Comment:
lift as an unexposed error?
(it's also used in `Imag` and `Real`, with same wording)

Reply:
I think that probably this and the other comment should be addressed in a sweep, this panic string exists in a variety of other functions as well.